newIDE/app/src/Export/Builds/BuildsList.js

```diff
@@ -2,106 +2,64 @@
 import { Trans } from '@lingui/macro';
 
 import * as React from 'react';
-import Paper from '@material-ui/core/Paper';
 import {
   type Build,
-  type BuildArtifactKeyName,
 } from '../../Utils/GDevelopServices/Build';
 import { Column, Line } from '../../UI/Grid';
 import EmptyMessage from '../../UI/EmptyMessage';
 import PlaceholderLoader from '../../UI/PlaceholderLoader';
-import BuildProgress from './BuildProgress';
 import { type AuthenticatedUser } from '../../Profile/AuthenticatedUserContext';
-import format from 'date-fns/format';
-import differenceInCalendarDays from 'date-fns/differenceInCalendarDays';
 import Text from '../../UI/Text';
+import { ColumnStackLayout } from '../../UI/Layout';
+import { BuildCard } from './BuildCard';
+import PlaceholderError from '../../UI/PlaceholderError';
 
 type Props = {|
   builds: ?Array<Build>,
   authenticatedUser: AuthenticatedUser,
-  onDownload: (build: Build, key: BuildArtifactKeyName) => void,
+  error: ?Error,
+  loadBuilds: () => void,
 |};
 
-const styles = {
-  buildContainer: {
-    padding: 10,
-  },
-};
-
-const formatBuildText = (
-  buildType: 'cordova-build' | 'electron-build' | 'web-build'
-) => {
-  switch (buildType) {
-    case 'cordova-build':
-      return <Trans>Android Build</Trans>;
-    case 'electron-build':
-      return <Trans>Windows/macOS/Linux Build</Trans>;
-    case 'web-build':
-      return <Trans>Web (upload online) Build</Trans>;
-    default:
-      return buildType;
-  }
-};
-
-export default ({ builds, authenticatedUser, onDownload }: Props) => {
+export default ({ builds, authenticatedUser, error, loadBuilds }: Props) => {
   return (
     <Column noMargin expand>
       <Line>
         <Column>
           <EmptyMessage>
             <Trans>
-              This is the list of builds that you've done. Note that you can
-              download games generated during the last 7 days, after which they
-              are removed.
+              This is the list of builds that you've done for this game. Note
+              that non-web buildsare available for 7 days, after which they are
```

Comment:
non-web builds => sounds technical.
Maybe something like "Note that builds for mobile or desktop can be downloaded for 7 days, after [...]".